src/com/reflections/RunLibrary2.java

```diff
@@ -0,0 +1,39 @@
+package com.reflections;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+/**
+ * * PART 2
+ * * В тому ж методі main зробити наступні дії:
+ * * Створити об’єкт класу Library не використовуючи конструктор.
+ * * Створити ще один об’єкт використовуючи конструктор.
+ * * В обох об’єктів замінити значення всіх полів, які мають модифікатор доступу private або final за допомогою інструментів reflection APІ.
+ */
+
+public class RunLibrary2 {
+
+    public static void main(String[] args) throws InvocationTargetException, InstantiationException, IllegalAccessException, NoSuchMethodException {
+
+        Library library = new Library("libka", 123, true, "etrte");
+        System.out.println(library + " - this is original Object from class Library");
+        System.out.println("--------------------------------------------------------------------------");
+
+        createObjectWithoutConstructor();
+        System.out.println("--------------------------------------------------------------------------");
+        createObjectByConstructor();
+    }
+
+    private static void createObjectWithoutConstructor() throws NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
```

Comment:
use try - catch, catch Exception ex, and re-throw a RuntimeException instead, all those ugly possible Exceptions aren't recommended. It was in Joshua Bloch book Effective Java he strongly suggested avoiding them and using non-checked exceptions (RuntimeException and it's children)